tests/test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -x
```

Comment:
This is VERY verbose. If we feel it needs to output something, it should provide very minimal output, one line per test, indicating test name and success or failure at best. `-x` mode is way too verbose to find actual failures ([~250 lines](https://travis-ci.org/voxpupuli/metadata-json-lint/jobs/172084400#L178-L425)).